system_tests/monitoring.py

```diff
@@ -175,3 +177,145 @@ def test_create_and_delete_metric_descriptor(self):
         descriptor.delete()
         with self.assertRaises(NotFound):
             descriptor.delete()
+
+
+class TestMonitoringGroups(unittest2.TestCase):
+
+    def setUp(self):
+        self.to_delete = []
+        self.DISPLAY_NAME = 'Testing: New group'
+        self.FILTER = 'resource.type = "gce_instance"'
+        self.IS_CLUSTER = True
+
+    def tearDown(self):
+        for group in self.to_delete:
+            backoff_intervals = [1, 2, 4, 8]
+            while True:
+                try:
+                    group.delete()
+                    break
+                except NotFound:
+                    if backoff_intervals:
+                        time.sleep(backoff_intervals.pop(0))
+                    else:
+                        raise
+
+    def test_create_group(self):
+        client = monitoring.Client()
+        group = client.group(
+            display_name=self.DISPLAY_NAME,
+            filter_string=self.FILTER,
+            is_cluster=self.IS_CLUSTER,
+        )
+        group.create()
+        self.to_delete.append(group)
+        self.assertTrue(group.exists())
```

Comment:
I have seen it pass it in the system tests I ran - but it probably is eventually consistent.
Any comments @rimey?